### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jacobsalmela**
Signed-off-by: jacobsalmela <jacob.salmela@pi-hole.net>

**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
---
**What does this PR aim to accomplish?:**
Introduce different issue templates as seen below.

screenshot 2018-08-12 16 59 03

Feedback is requested and expected to get a result we can all agree on.

**How does this PR accomplish the above?:**
Adds new issue template files.

**What documentation changes (if any) are needed to support this PR?:**
None.

### CODE REVIEW

**Comment by Mcat12** [comment]
@jacobsalmela try force-pushing an amended commit with the DCO sign-off.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (Mcat12) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This template is very similar to the previous one (`bug-report.md`), but has less information. Can it be removed?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+** Please respect that **Pi-hole is developed by volunteers**, who can only reply in their spare time, so you're adherence to our template will help make the most of our limited time.
```
[the remark this answers is not in the document]
Comment (dschaper) [reply .github/ISSUE_TEMPLATE/bug-report.md]: @PromoFaux would have to tweak Gerald to strip out the comment lines.

**Comment by dschaper** [comment]
Let's start this again later after some changes are made to the repository.

**Pull request closed without merging**